org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/management/RepositoryResolverFactory.java

```diff
@@ -33,8 +33,8 @@
 
     public static Optional<IRepositoryResolver> getResolver(String url, String branch) {
         try {
-            if (new URL(url).getHost().contains("github.com")) {
-                return Optional.of(new GitHubResolver(url, branch));
+            if (new URL(url).getHost().contains("github.com") || new URL(url).getHost().contains("gitlab")) {
```

Comment:
this still precludes support for other git providers, such as bitbucket or on-premise hosting for solutions like GitHub Enterprise.

It would be best to not assume the URL contains any identifiers ...